### PR TITLE
17795 Allow registration (SP/GP) for an user/account that only uses last name

### PR DESCRIPTION
**Pull request by chenhongjing**
*Issue #:* /bcgov/entity#17795

*Description of changes:*
- Removed invalid first name error
- Changed first name rule of completing party for registration
- Changed `certifiedBy` value
- app version = 5.5.2

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-create-dev--pr-569-w8e4eyhb.web.app

File: src/rules/first-name-rules.ts [hunk 1]
```diff
@@ -6,6 +6,5 @@ export const FirstNameRules: Array<VuetifyRuleFunction> = [
 ]
 
 export const FirstNameRulesFirms: Array<VuetifyRuleFunction> = [
-  v => !!v?.trim() || 'First name is required',
```
Comment (severinbeauvais) [inline src/rules/first-name-rules.ts:9]: Is this rule used only for Completing Party?

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-create-dev--pr-569-w8e4eyhb.web.app

File: src/components/common/RegAddEditOrgPerson.vue [hunk 2]
```diff
@@ -78,7 +78,9 @@
                   filled
                   class="item first-name"
                   label="First Name"
-                  :rules="enableRules ? Rules.FirstNameRulesFirms : []"
+                  :rules="enableRules ?
+                   (isCompletingParty ? Rules.FirstNameRulesFirms.slice(1) :
+                   Rules.FirstNameRulesFirms) : []"
```
Comment (severinbeauvais) [inline src/components/common/RegAddEditOrgPerson.vue:83]: 1. This is too much logic for the template, but...
2. This is not a good way to implement this as it assumes that the first rule (which you are omitting) is for first name. However, someone could change the rules (in the other file) and would not know to update this file, causing a bug.

I recommend creating a different rule for Completing Party. You could add it to the same file as the other rule.

Comment (severinbeauvais) [reply src/components/common/RegAddEditOrgPerson.vue:83]: Thanks for the fix.

File: src/rules/first-name-rules.ts (hunk 1 above)
Comment (severinbeauvais) [reply src/rules/first-name-rules.ts:9]: No longer relevant.

**Comment by severinbeauvais** [comment]
Please fix lint errors and unit test.

**Review by severinbeauvais — APPROVED** [review]

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-create-dev--pr-569-w8e4eyhb.web.app

**Review by JazzarKarim — APPROVED** [review]

**Pull request merged**